src/main/java/com/internet/shop/service/impl/OrderServiceImpl.java

```diff
@@ -44,4 +46,11 @@ public Order get(Long id) {
     public boolean delete(Long id) {
         return orderDao.deleteById(id);
     }
+
+    @Override
+    public BigDecimal findSum(Long orderId) {
```

Comment:
let's pass order to this method so we will not call database a second time

Reply:
Sure